### ISSUE TITLE
Improve tying up framework

### ISSUE THREAD

**Issue by DanisAlukaev** [DanisAlukaev/XCBs#48]
It seems that framework yields poor results due to smaller logits on textual part. Here are some possible solutions:
1. Instead of symmetrical learning, pre-train embeddings and use them to approximate logits on visual part;
2. Make captions of dataset even more difficult;
3. Use MAE as tie loss.
